els/accounts/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path, include
+from .api import RegisterAPI, LoginAPI, UserAPI
+from knox import views as knox_views
+
+urlpatterns = [
+  path('api/auth', include('knox.urls')),
+  path('api/auth/register', RegisterAPI.as_view()),
+  path('api/auth/login', LoginAPI.as_view()),
+  path('api/auth/user', UserAPI.as_view()),
+  path('api/auth/logout', knox_views.LogoutView.as_view(), name='knox_logout')
```

Comment:
Remove `api/` as it will be handled by the parent `urls.py` wherein you will put the `api/` prefix
```suggestion
  path('auth', include('knox.urls')),
  path('auth/register', RegisterAPI.as_view()),
  path('auth/login', LoginAPI.as_view()),
  path('auth/user', UserAPI.as_view()),
  path('auth/logout', knox_views.LogoutView.as_view(), name='knox_logout')
```